geometry/jarvis_algorithm.cpp

```diff
@@ -120,16 +119,18 @@ namespace geometry {
             }
 
             /**
-             *
+             * This function returns the geometric orientation for the three points
+             * in a space, ie, whether they are linear ir clockwise or
+             * anti-clockwise
              * @param p first point selected
              * @param q adjacent point for q
              * @param r adjacent point for q
-             * @returns the geometric orientation for the three points
```

Comment:
you can keep this line along with the other return statements

Reply:
I just moved this line to description and added @returns to the three values